float32[7] stiffness
float32[7] damping